ci/azure-pipelines.yml

```diff
@@ -31,6 +31,16 @@ jobs:
       - script: bash <(curl https://codecov.io/bash)
         displayName: Upload coverage to Codecov
 
+  - job: BDDTest
+    pool:
+      vmImage: ubuntu-16.04
+    timeoutInMinutes: 30
+    steps:
+      - template: azp-dependencies.yml
+      - checkout: self
+      - script: make bdd-test
```

Comment:
you need to login to github packages

Reply:
https://github.com/trustbloc/edge-sandbox/blob/master/ci/azure-pipelines.yml#L51

Reply:
done